scripts/install

```diff
@@ -15,4 +15,5 @@ else
     PIP="pip"
 fi
 
+"$PIP" install -r "requirements-pypa.txt"
```

Comment:
Perhaps let's just do...?

```
"$PIP" install -U pip wheel setuptools
```

For the sake of simplicity, if that's what we need as a prerequisite.

Reply:
this is just for debugging locally - `python3.6 -m venv` creates an environment that can't install `cryptography`